otelzap/otelzap.go

```diff
@@ -439,7 +450,7 @@ func (s *SugaredLogger) FatalfContext(ctx context.Context, template string, args
 func (s *SugaredLogger) logArgs(
 	ctx context.Context, lvl zapcore.Level, template string, args []interface{},
 ) {
-	if lvl < s.l.minLevel {
+	if lvl <= s.l.minLevel {
```

Comment:
The current code for this line is causing some weird behaviors when the log level is `debug`